docs/source/system_configs.rst

```diff
@@ -17,7 +17,7 @@ are overlapping keys in different files the last file will win.
 If a file has permissions that prevent us from reading it, then that file will be ignored.
 This is useful for credentials that only some users or scripts need access to.
 
-See `load_system_paasta_config <generated/paasta_tools.utils.html#paasta_tools.utils.load_system_paasta_config>`_ for
+See `load_system_PaaSTA_config <generated/PaaSTA_tools.utils.html#PaaSTA_tools.utils.load_system_PaaSTA_config>`_ for
```

Comment:
same as in a previous file - the links should probably stay lower-case (and the functions are all lowercase)